desc/integrals/_bounce_utils.py

```diff
@@ -673,22 +818,22 @@ def interp_to_argmin_hard(h, points, knots, g, dg_dz, method="cubic"):
     Parameters
     ----------
```

Comment:
in the docstring,what is P>>1 mean? what is P?

Reply:
please change `P` to `num pitch`